app/src/main/java/org/sopt/and/signin/SignInScreen.kt

```diff
@@ -0,0 +1,154 @@
+package org.sopt.and.signin
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.R
+import org.sopt.and.component.DividerWithText
+import org.sopt.and.component.OtherServiceIconRow
+import org.sopt.and.component.WavveActionTextField
+import org.sopt.and.component.WavveTextField
+import org.sopt.and.signin.model.SignInState
+import org.sopt.and.ui.theme.SecondGrey
+import org.sopt.and.ui.theme.ThirdGrey
+
+@Composable
+fun SignInScreen(
+    state: SignInState,
+    onIdChange: (String) -> Unit,
+    onPasswordChange: (String) -> Unit,
+    onSignInButtonClick: () -> Unit,
+    onFindInButtonClick: () -> Unit,
+    onPasswordResetButtonClick: () -> Unit,
+    onSignUpButtonClick: () -> Unit,
+    modifier: Modifier
+){
+    Column(
+        modifier = modifier.fillMaxSize()
+            .padding(top = 48.dp)
+    ){
+        WavveTextField(
+            value = state.id,
+            hint = stringResource(R.string.signup_login_hint),
+            onValueChange = onIdChange,
+            modifier = Modifier.padding(horizontal = 8.dp)
+        )
+
+        Spacer(modifier = Modifier.height(4.dp))
+
+        WavveActionTextField(
+            value = state.password,
+            hint = stringResource(R.string.signin_password_hint),
+            onValueChange = onPasswordChange,
+            modifier = Modifier.padding(horizontal = 8.dp)
+        )
+
+        Spacer(modifier = Modifier.height(32.dp))
+
+        Button(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 8.dp)
+            ,
+            onClick = {
+                onSignInButtonClick()
+            },
```

Comment:
인자가 없는 경우, 혹은 함수의 인자 순서와 람다의 변수 순서가 같다면 람다 없이 함수이름만 적으셔도 됩니다!
가독성이 더 좋아지겠죠?
```suggestion
            onClick = onSignInButtonClick,
```

Reply:
알고 있던 사실인데도 놓쳤네요 ㅎㅎ 감사합니다